src/Database/Adapter/MariaDB.php

```diff
@@ -878,8 +878,11 @@ public function find(string $collection, array $queries = [], int $limit = 25, i
             });
 
             $conditions = [];
+            $attributeIndex = 0;
             foreach ($query->getValues() as $key => $value) {
-                $conditions[] = $this->getSQLCondition('table_main.`' . $query->getAttribute().'`', $query->getMethod(), ':attribute_' . $i . '_' . $key . '_' . $query->getAttribute(), $value);
+                $bindKey = 'key_' . $attributeIndex;
+                $conditions[] = $this->getSQLCondition('table_main.`' . $query->getAttribute() . '`', $query->getMethod(), ':attribute_' . $i . '_' . $key . '_' . $bindKey, $value);
+                $attributeIndex++;
```

Comment:
We can use a post-increment here to make the intention clear on the same line as the usage:

```suggestion
                $bindKey = 'key_' . $attributeIndex++;
                $conditions[] = $this->getSQLCondition('table_main.`' . $query->getAttribute() . '`', $query->getMethod(), ':attribute_' . $i . '_' . $key . '_' . $bindKey, $value);
```

Applies to everywhere we're using `$attributeIndex` in this PR

Reply:
Actually I see now the way you've done it is consistent with the rest of the codebase, resolving this comment :+1: